spec/forms/hyrax/forms/resource_form_spec.rb

```diff
@@ -384,7 +384,13 @@
     end
 
     context 'when using a generic valkyrie adapter', valkyrie_adapter: :test_adapter do
+      before do
+        allow(Hyrax).to receive_message_chain(:config, :disable_wings).and_return(true) # rubocop:disable RSpec/MessageChain
+        hide_const("Wings") # disable_wings=true removes the Wings constant
+      end
       it 'prepopulates as empty before save' do
+        expect(Hyrax.logger).to receive(:info)
+          .with(starting_with("trying to prepopulate a lock token for"))
```

Comment:
It isn't a change that the logger adds this info message.  This expect line just confirms the message was logged.